gef-ui/server/controllers/application-details/index.js

```diff
@@ -151,20 +163,36 @@ const applicationDetails = async (req, res, next) => {
       maker,
     };
 
-    const partial = stateToPartial(application.status, req.url);
-    let params;
+    if (req.url) {
+      url = req.url.split('/');
+      url = url[url.length - 1];
+    }
+
+    if (facilitiesPartials.includes(url)) {
```

Comment:
what should happen if it's not `confirm-cover-start-date` or `ukef-cover-start-date`?

Reply:
Then the facilities object is not send as a param to the rending njk template and respective template will be loaded based on application status.